clarity_ext/repository/step_repository.py

```diff
@@ -67,26 +62,35 @@ def all_artifacts(self):
         # that we create only one artifact domain object in this case:
         outputs_by_id = dict()
         container_repo = ContainerRepository()
-        for input_res, output_res in input_output_maps:
+        for genologics_input, genologics_output, output_generation_type in wrappable_pairs:
```

Comment:
Could we rename `genologics_` to e.g. `rest_`, `api_` or `raw_`? I think we never refer to it as `genologics_` - probably just `api_`.